static/src/javascripts/bootstraps/common.js

```diff
@@ -548,6 +548,9 @@ define([
             robust('c-run-custom-ab',   modules.runCustomAbTests);
             robust('c-accessibility-prefs',       modules.initAccessibilityPrefs);
             robust('c-international-signposting', modules.internationalSignposting);
+            if (window.console && window.console.log) {
```

Comment:
I'd put this behind a localStorage switch se that we can disable it. ~99% of people looking at the console is already a guardian developer